docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+
+Conference websites
+===================
+
+Each conference has a section of the website, which is part of the same
+repository as the main conference website. Many parts of the conference
+websites are data-driven, like the generation of the schedule.
+
+.. note::
+    This documentation applies only for conferences from 2020 onwards.
+
+File locations
+--------------
+
+The important locations of files are:
+
+* ``_data/<shortcode>-<year>-config.yaml`` contains the general config file
+  for rendering a specific conference. This includes settings like the
+  conference dates, sponsors and ticket prices.
+* ``_data/<shortcode>-<year>-sessions.yaml`` contains the conference sessions,
+  the main talks on the agenda. For each session, it includes things like the
+  title and abstract, and name and other info for the speaker(s).
+* ``_data/<shortcode>-<year>-schedule.yaml`` contains the conference schedule.
+  This includes the talks, but also items like "Lunch".
+* ``conf/<shortcode>/<year>/`` contains the RST files used for rendering the
+  conference website. Copy these over from the previous conference
+  (chronologically, not the previous conference in the same location).
+  You may need to update some reference to specific locations/years,
+  and keep an eye on any FIXME's or TODO's.
+* ``include/conf/`` contains text snippets which are generally the same
+  between conferences, or at least between conferences in the same location.
+  Examples include the description of lightning talks, or the Portland hike.
+* ``_templates/<year>/`` and ``_templates/include/`` contain HTML or RST
+  templates used in rendering the conference pages. This includes the
+  website design, navigation and snippets used in schedule rendering.
+
+The YAML files are validated using the schemas in ``_data/schema-*``.
+
+
+Typical conference workflow
+---------------------------
+
+* The initial conference site is created, with ``flaglanding`` typically
+  set to publish a simple landing page with the conference dates.
+* At some point, the content for the conference is filled in and verified,
+  like sponsorship prices. ``flaglanding`` is now switched off and remains off.
+* As soon as tickets go on sale, ``flagticketsonsale`` is switched on.
+* When the CFP opens, ``flagcfp`` is switched on. Once the CFP has closed,
+  it is switched off and remains off. The CFP details should be in the
+  general conference config file, under ``cfp``.
+* When speakers are announced, ``flagspeakersannounced`` is switched on
+  and remains on. The sessions should be in the
+  ``_data/<shortcode>-<year>-sessions.yaml`` file.
+* When the schedule is ready, ``flaghasschedule`` is switched on and remains on.
+  The schedule should be in the ``_data/<shortcode>-<year>-schedule.yaml`` file.
+* At some point the conference sells out. ``flagticketsonsale`` is switched off,
+  ``flagsoldout`` is switched on.
+* During the conference, if live streaming is available, ``flaglivestreaming``
```

Comment:
This needs to happen before the conference, not during, so that we can promote the live stream (1-2 weeks?)